onset_maproom/maproom.py

```diff
@@ -82,6 +84,37 @@ def map_click(click_lat_lng):
         children=dlf.Tooltip("({:.3f}, {:.3f})".format(*lat_lng))
     )
 
+@APP.callback(
+    Output("plotly_onset_test", "figure"),
+    Input("map", "click_lat_lng"),
+    Input("earlyStartDay", "value"),
+    Input("earlyStartMonth", "value"),
+    Input("searchDays", "value"),
+    Input("wetThreshold", "value"),
+    Input("runningDays", "value"),
+    Input("runningTotal", "value"),
+    Input("minRainyDays", "value"),
+    Input("dryDays", "value"),
+    Input("drySpell", "value"),
+)
+def onset_plot(click_lat_lng, earlyStartDay, earlyStartMonth, searchDays, wetThreshold, runningDays, runningTotal, minRainyDays, dryDays,drySpell):
+    lat, lng = get_coords(click_lat_lng)
+    ds = rr_mrg.sel(X=lng, Y=lat, method="nearest")
```

Comment:
Minor suggestion: add `drop=True` to get rid of the X and Y dimensions here. While stepping through the code, I noticed that the `onsetMD` dataframe currently looks like this:
```
                  X       Y  Onset Date
T                                      
1981-06-01  38.7375  9.0375  1999-08-07
1982-06-01  38.7375  9.0375  1999-06-18
1983-06-01  38.7375  9.0375  1999-08-05
```

Reply:
Okay. Is there a benefit for doing this other than it making the data more simple?

Reply:
Probaby no advantage to doing it in general. Ordinarily it takes very little extra space to keep the extra dimensions, and the extra metadata might come in handy sometimes. The problem I noticed only arises when you use `to_dataframe`, which copies the metadata into each row of the table. For a table with many rows, this could waste a significant amount of space. (For this application, where the number of rows is pretty small, it really makes no difference. I probably shouldn't have mentioned it.)